checkov/kustomize/runner.py

```diff
@@ -98,13 +99,16 @@ def mutateKubernetesGraphResults(self, root_folder: str, runner_filter: RunnerFi
         return report
 
 class Runner(BaseRunner):
-    check_type = CheckType.KUSTOMIZE
     kustomize_command = 'kustomize'
+    kubectl_command = 'kubectl'
+    check_type = CheckType.KUSTOMIZE
     system_deps = True
     potentialBases = []
     potentialOverlays = []
     kustomizeProcessedFolderAndMeta = {}
     kustomizeFileMappings = {}
+    kustomizeSupportedFileTypes = ['kustomization.yaml','kustomization.yml']
```

Comment:
```suggestion
    kustomizeSupportedFileTypes = ('kustomization.yaml', 'kustomization.yml')
```
if you define a static list it is always preferable to use a tuple, because they more memory efficient and have better performance when used as a member check compared to a list.

Reply:
Thanks!